warehouse/query-core/src/main/java/datawave/query/iterator/QueryOptions.java

```diff
@@ -958,6 +961,14 @@ public void setIvaratorCacheBufferSize(int ivaratorCacheBufferSize) {
         this.ivaratorCacheBufferSize = ivaratorCacheBufferSize;
     }
 
+    public int getUniqueCacheBufferSize() {
```

Comment:
`getUniqueCacheBufferSize()` and `setUniqueCacheBufferSize()` should be deleted.